app/service.py

```diff
@@ -44,53 +90,50 @@ async def startup():
 async def shutdown():
     await db.disconnect()
 
-
-@app.post("/api/v1/providers", response_model=schemas.APIKey)
+@app.post("/api/v1/providers", response_model=schemas.APIKey, tags=["Requests with apiKey 🔑"])
```

Comment:
It would be better to have a new "definitions" file, where a TAG Enum can be defined for these, instead of copying the same string in many places.

Reply:
Buenas corregido con un enum dentro de model:

![image](https://github.com/tchx84/linka/assets/57275367/c5079a5f-047a-4ed0-add0-0af27b4bb703)

y se utiliza en los endpoints  así:
![image](https://github.com/tchx84/linka/assets/57275367/ee0e2360-b911-4f42-bb7c-9ef209894f75)